services-camel/src/main/resources/config.properties

```diff
@@ -9,17 +9,23 @@ fcrepo.authPassword=
 fcrepo.authHost=
 
 cdr.enhancement.thumbnail.fileExtension=PNG
+cdr.enhancement.thumbnail.mimetype=image/png
 cdr.enhancement.fulltext.fileName=full_text.txt
+cdr.enhancement.jp2.fileExtension=JP2
+cdr.enhancement.jp2.mimetype=image/jp2
+cdr.enhancement.postIndexingDelay=5000
+cdr.enhancement.processingThreads=2
 
 # In the event of failure, the maximum number of times a redelivery will be attempted.
```

Comment:
Would you mind specifying units on the retryDelay in the line comment, just to help any unwitting dev who looks at this in future?